src/components/ha-fab.ts

```diff
@@ -11,11 +11,9 @@ export class HaFab extends Fab {
 
   static override styles = Fab.styles.concat([
     css`
-      :host-context([style*="direction: rtl;"])
-        .mdc-fab--extended
-        .mdc-fab__icon {
-        margin-left: 12px !important;
-        margin-right: calc(12px - 20px) !important;
+      .mdc-fab--extended .mdc-fab__icon {
+        margin-left: var(--rtl-12px, calc(12px - 20px)) !important;
```

Comment:
Can we use `margin-inline-start` and `margin-inline-end` here instead of `margin-left` and `margin-right`?

Reply:
And set `direction: var(--dir);`

Reply:
Set left and right to 0?

Reply:
But that's not what MWC does so I think it is better to patch it the opposite to the original for better maintenance

Reply:
It would be the same? It would be easier to maintain because we can use the same for ltr and rtl

Reply:
Yes we write margin-left: var(--rtl-12px, -8px) !important;
Which reads: var(RTL VALUE, LTR VALUE)

Reply:
I still think using `margin-inline-start` would be a better option, as it doesn't require us to set all these variables and make it extra complicated

Reply:
I tried inline-start, etc'... We can't use it since looking at MWC - the direction of the elements is LTR even if HA is in RTL so it calculated it backward. This will lead us to endless problems that I don't know how to get out of.